src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: gray;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: white;
+}
+
+.logo__item {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
```

Comment:
Always remove unwanted default styles using a reset or normalizer. Here is reset
https://meyerweb.com/eric/tools/css/reset/